drop-index.md

```diff
@@ -14,12 +14,36 @@ The `DROP INDEX` [statement](sql-statements.html) removes an index from a table.
 
 ## Required Privileges
 
-The user must have the `CREATE` [privilege](privileges.html) on the table. 
+The user must have the `CREATE` [privilege](privileges.html) on each specified table.
+
 
 ## Parameters
 
 | Parameter | Description |
 |-----------|-------------|
-|  |  |
-
-
+| `IF EXISTS`	| Drop the named indexes if they exist; if they do not exist, do not return an error.|
+| `table_name`	| The name of the table with the index you want to drop. Find table names with [`SHOW TABLES`](show-tables.html).|
+| `index_name`	| The name of the index you want to drop. Find index names with [`SHOW INDEX`](show-index.html).<br/><br/>You cannot drop a table's `primary` index.|
```

Comment:
Some elements apply only to one table/index. Others can apply to multiple, so they're pluralized. Sincerely, though, appreciate the attention to detail.